webui/package.json

```diff
@@ -17,7 +17,7 @@
     "prepare-e2e": "cypress/scripts/pass-gen.sh && ts-node cypress/scripts/prepareData.ts",
     "cleanup": "ts-node cypress/scripts/cleanup.ts",
     "report-coverage": "codecov",
-    "init-admin": "ts-node cypress/scripts/init-admin.ts"
+    "init-admin": "cypress/scripts/pass-gen.sh && ts-node cypress/scripts/init-admin.ts"
```

Comment:
`pass-gen.sh` creates `credentials.js`, should it be creating `credentials.ts`?

Reply:
either is fine. The tsconfig is set to `allowJS`. I took the js one as it's easier when we ask users to create it manually. Now it's with the script, it's the same either way I think. But never bother to change it as it works just fine.

Reply:
Cool thanks.